fluentd/generate_throttle_configs.rb

```diff
@@ -102,8 +102,9 @@ def create_default_docker(excluded)
 
   file_name = "/etc/fluent/configs.d/input/docker/default-docker.conf"
 
-  File.open(file_name, 'w') { |file|
-    file.write(<<-CONF)
+  unless ENV['USE_JOURNAL'] == "true"
```

Comment:
Yeah, that's a big problem.  I'm not sure what to do about that since we can't throttle from journal (in fluentd, anyway).